src/tree.ml

```diff
@@ -201,6 +201,8 @@ module CP = struct
     if res = None then choose_vf pools else res
 end
 
+(** Prefix of bound variables. *)
+let var_prefix = "var"
```

Comment:
Can probably be removed once the tvar arguments are removed from the tree type.

Reply:
I propose to remove this definition as there is no reason to share it between tree.ml and tree_graphviz.ml.